packages/maskbook/src/plugins/RedPacket/types.ts

```diff
@@ -27,13 +28,18 @@ export enum RedPacketStatus {
     refunded = 'refunded',
 }
 
+export enum DialogTabs {
+    create = 0,
+    past = 1,
+}
+
 export interface RedPacketAvailability {
     token_address: string
     balance: string
     total: string
     claimed: string
     expired: boolean
-    ifclaimed: boolean
+    claimed_amount: string
```

Comment:
is it a breaking change?

Reply:
yes, already handled it: https://github.com/DimensionDev/Maskbook/blob/8accb67a36f00124011538805797cdbf0aa6651e/packages/maskbook/src/plugins/RedPacket/hooks/useAvailabilityComputed.ts#L32

Reply:
I mean how do you handle the old name `ifclaimed`

Reply:
`const isClaimed = availability.ifclaimed` => `const isClaimed = availability.claimed_amount !== '0' `

Reply:
So IIRC the field "ifclaimed" is actually used in the payload. did it changed too?

Reply:
`ifclaimed` does not exist in payload